### ISSUE TITLE
Extract code from markdown file

### ISSUE THREAD

**Issue by vincentqb** [aaren/notedown#37]
Is there a way of extracting the code chunks directly from notedown?

**Comment by aaren** [comment]
You mean to strip all of the non code content from a file?

There isn't a way to do that but it wouldn't be difficult to add. It would have to be enabled with a flag. What do you think a good flag name would be?

**Comment by vincentqb** [comment]
Yes, either stripping or commenting out the non-code content would do. The later option is what ipython nbconvert does when converting to code.

For the flag, how about "--to code" or "--code" ?

**Comment by randy3k** [comment]
I suggest to support something like knitr-spin
https://github.com/yihui/knitr/blob/master/inst/examples/knitr-spin.R

Basically, it converts the markdown blocks to special comment lines (with `#'` starts) and the code block to plain code lines. 

And it is more interesting to support the another direction of conversion. Documented python, jupyter or R scripts could be easily converted to jupyter notebooks.
